.github/workflows/testBuild.yml

```diff
@@ -206,6 +193,8 @@ jobs:
           "Use the links below to test this build in android and iOS. Happy testing!
           | android :robot:  | iOS :apple: | 
           | ------------- | ------------- |
-          | ${{fromJson(steps.set_var.outputs.android_paths).html_path}}  | ${{fromJson(steps.set_var.outputs.ios_paths).html_path}}  |"
+          | ${{fromJson(steps.set_var.outputs.android_paths).html_path}}  | ${{fromJson(steps.set_var.outputs.ios_paths).html_path}}  |
+          | ------------- | ------------- |
```

Comment:
This ends up rendering kind of weird:

<img width="492" alt="image" src="https://user-images.githubusercontent.com/47436092/204882746-d2c5c025-1436-42cb-958e-c44963112801.png">

```suggestion
```

Reply:
Also, why only Android and iOS instead of all platforms?

Reply:
Thanks for the suggestion! We wanted to make sure that mobile part works first, and I am not 100% sure where exactly `.dmg` file is going to land on S3. I've updated the file, but there is a slight chance it is not going to be the right path. As per the web app there is a small problem with hosting multiple react apps in one bucket, it seems that it is not as easy as it seemed at the beginning.